lib/active_merchant/billing/gateway.rb

```diff
@@ -275,14 +275,14 @@ def localized_amount(money, currency)
         if non_fractional_currency?(currency)
           if self.money_format == :cents
             sprintf("%.0f", amount.to_f / 100)
-          else
+          elsif self.money_format == :dollars
```

Comment:
I don't think it has a logical impact. I was just curious that maybe I was missing something. 

I think style wise I tend to say just use an `else` if there are only two options. Interestingly enough I would probably feel a little less clear since we call the money format "dollars" (which really just means "with a decimal") when the currency in question isn't actually called a "dollar".